Packs/Mimecast/Integrations/MimecastV2/MimecastV2.yml

```
@@ -1631,15 +1748,802 @@ script:
     - contextPath: Mimecast.Policy.Receiver.Type
       description: The receiver type.
       type: String
+  - name: mimecast-search-message
+    description: "Searches a message "
+    deprecated: false
+    arguments:
+    - name: search_reason
+      description: Reason for tracking the email
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: from_date
+      description: Api start parameter. Datetime format is ISO 8601. e.g. 2022-07-11T14:49:18+0000.
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: to_date
+      description: Api end parameter  Datetime format ISO 8601. e.g. 2022-07-11T14:49:18+0000.
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: message_id
+      description: The internet message id of the message to track.
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: from
+      description: "Part of advancedTrackAndTraceOptions object : The sending email address or domain of the messages to track."
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: to
+      description: "Part of advancedTrackAndTraceOptions object: The recipient email address or domain of the messages to track."
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: subject
+      description: "Part of advancedTrackAndTraceOptions object : The subject of the messages to track."
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: sender_ip
+      description: "Part of advancedTrackAndTraceOptions object : The source IP address of messages to track."
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: route
+      description: An array of routes to filter by. Possible values are internal, outbound and inbound.
```

```suggestion
      description: A comma-separated list of routes to filter by. Possible values are internal, outbound and inbound.
```
let's check if `Possible values are internal, outbound and inbound.` is not auto genetated in the UI